Packs/TwitterIOCHunter-FullDailyFeed/pack_metadata.json

```diff
@@ -10,7 +10,9 @@
     "categories": ["Data Enrichment & Threat Intelligence"],
     "tags": [
         "Threat Intelligence Management",
-        "Getting Started"
+        "Getting Started",
```

Comment:
As its community supported, lets not change this pack metadata.
I also know that Twitter API changed recently (with paid version).

Reply:
Ok. but it has "Getting Started"